front50-protobuf/front50-protobuf.gradle

```diff
@@ -0,0 +1,45 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.8'
+  }
+}
+
+apply plugin: 'java'
+apply plugin: 'com.google.protobuf'
+
+ext {
+  protobufVersion = '3.2.+'
+}
+
+dependencies {
+  api "com.google.protobuf:protobuf-java:${protobufVersion}"
```

Comment:
I _think_ the protobuf version is provided by spinnaker-dependencies? Even if it isn't, I think it should be so we don't have individual repos defining this.